openapi_python_client/templates/endpoint_macros.pyi

```diff
@@ -33,11 +33,12 @@ params: Dict[str, Any] = {
 }
     {% for property in endpoint.query_parameters %}
         {% if not property.required or property.nullable %}
-if {% if not property.required %}{{ property.python_name }} is not UNSET and {% endif %}{{ property.python_name }} is not None:
-            {% if property.template %}
-    params["{{ property.name }}"] = {{ "json_" + property.python_name }}
+            {% set property_name = "json_" + property.python_name if property.template else property.python_name %}
+if {% if not property.required %}not isinstance({{ property_name }}, Unset) and {% endif %}{{ property_name }} is not None:
+            {% if property.is_dict %}
+    params.update({{ property_name }})
```

Comment:
Does this do the prefixing as expected?

I think I would expect something like
```suggestion
    params.update({
    	f"{property.name}.{p}": v, for p, v in {{ property_name }}.items()
    })
```
in order for it to build the params with e.g. `schemaField.Plasmid=seq_XYZ` instead of just `Plasmid=seq_XYZ`

Reply:
It does not handle the prefixing. According to the OpenAPI spec, the parameter name itself does not show up at all in the serialization.

See https://swagger.io/specification/ under "Style Examples"